src/server/admin_realm.hpp

```diff
@@ -22,6 +22,7 @@
 #include "results.hpp"
 #include "shared_realm.hpp"
 #include "sync/sync_config.hpp"
+#include "realm/object_id.hpp"
```

Comment:
Don't need this header; just need to move the forward declaration of ObjectID out of the sync namespace.